src/analysis/normed_space/operator_norm.lean

```diff
@@ -292,39 +292,27 @@ norm_id ⟨1, one_ne_zero⟩
 @[simp] lemma norm_id_field' : ∥(1 : 𝕜 →L[𝕜] 𝕜)∥ = 1 :=
 norm_id_field
 
-/-- The operator norm is homogeneous. -/
-lemma op_norm_smul : ∥c • f∥ = ∥c∥ * ∥f∥ :=
-le_antisymm
-  ((c • f).op_norm_le_bound
-    (mul_nonneg (norm_nonneg _) (op_norm_nonneg _)) (λ _,
-    begin
-      erw [norm_smul, mul_assoc],
-      exact mul_le_mul_of_nonneg_left (le_op_norm _ _) (norm_nonneg _)
-    end))
-  (lb_le_Inf _ bounds_nonempty (λ _ ⟨hn, hc⟩,
-    (or.elim (lt_or_eq_of_le (norm_nonneg c))
-      (λ hlt,
-        begin
-          rw mul_comm,
-          exact mul_le_of_le_div hlt (Inf_le _ bounds_bdd_below
-          ⟨div_nonneg hn hlt, λ _,
-          (by { rw div_mul_eq_mul_div, exact le_div_of_mul_le hlt
-          (by { rw [ mul_comm, ←norm_smul ], exact hc _ }) })⟩)
-        end)
-      (λ heq, by { rw [←heq, zero_mul], exact hn }))))
-
-lemma op_norm_neg : ∥-f∥ = ∥f∥ := calc
-  ∥-f∥ = ∥(-1:𝕜) • f∥ : by rw neg_one_smul
-  ... = ∥(-1:𝕜)∥ * ∥f∥ : by rw op_norm_smul
-  ... = ∥f∥ : by simp
+lemma op_norm_smul_le : ∥c • f∥ ≤ ∥c∥ * ∥f∥ :=
+((c • f).op_norm_le_bound
+  (mul_nonneg (norm_nonneg _) (op_norm_nonneg _)) (λ _,
+  begin
+    erw [norm_smul, mul_assoc],
+    exact mul_le_mul_of_nonneg_left (le_op_norm _ _) (norm_nonneg _)
+  end))
+
+lemma op_norm_neg : ∥-f∥ = ∥f∥ :=
+begin
+  show Inf { c | c ≥ 0 ∧ ∀ x, ∥(-f) x∥ ≤ c * ∥x∥ } = Inf { c | c ≥ 0 ∧ ∀ x, ∥f x∥ ≤ c * ∥x∥ },
```

Comment:
```suggestion
  show Inf {c | c ≥ 0 ∧ ∀ x, ∥(-f) x∥ ≤ c * ∥x∥} = Inf {c | c ≥ 0 ∧ ∀ x, ∥f x∥ ≤ c * ∥x∥},
```

Reply:
Sure.  By the way, can this line be shortened?  (I wrote it.)  I needed to unfold the definition of the `op_norm` into this `Inf` expression, so that the next steps work.  Is there a way to tell it, "rewrite using the definition", without explicitly writing out the expressions that should result?

Reply:
If the definition is not hidden behind an instance, then you can `rw [myfunction]`. In this case the easiest way is to add `lemma norm_def` and rewrite on it.

Reply:
Thanks, I made this improvement in b3b8a79